src/code/code_800F7260.c

```diff
@@ -60,34 +60,34 @@ u8 D_801333CC = 0;
 
 u16 D_801333D0 = 0;
 
-Vec3f D_801333D4 = { 0.0f, 0.0f, 0.0f };
+Vec3f D_801333D4 = { 0.0f, 0.0f, 0.0f }; // default pos
 
-f32 D_801333E0 = 1.0f;
+f32 D_801333E0 = 1.0f; // default freqScale
 
-UNK_TYPE D_801333E4 = 0;
+UNK_TYPE D_801333E4 = 0; // unused
 
-s8 D_801333E8 = 0;
+s8 D_801333E8 = 0; // default reverbAdd
 
-UNK_TYPE D_801333EC = 0;
+UNK_TYPE D_801333EC = 0; // unused
 
 u8 D_801333F0 = 0;
 
-u8 D_801333F4 = 0;
+u8 gAudioSEFlagSwapOff = 0;
 
 u8 D_801333F8 = 0;
 
-extern Struct_800F738C D_8016C9A0[];
-extern Struct_800F8EA0 D_8016E270[];
+extern SoundRequest sSoundRequests[0x100];
+extern Struct_800F8EA0 D_8016E270[7];
 extern Struct_8013331C* D_8013331C[9];
 
-void func_800F7260(u16 arg0) {
+void Audio_MuteSoundBanks(u16 arg0) {
     u8 bankId;
 
     for (bankId = 0; bankId < ARRAY_COUNT(gSoundBanks); bankId++) {
         if (arg0 & 1) {
-            D_8016E264[bankId] = 1;
+            gSoundBankMuted[bankId] = true;
         } else {
-            D_8016E264[bankId] = 0;
+            gSoundBankMuted[bankId] = false;
         }
         arg0 = (arg0 >> 1) & 0xFFFF;
```

Comment:
```suggestion
        arg0 = arg0 >> 1;
```
for whatever reason `arg0 >>= 1;` doesn't match